python/hsfs/feature_group.py

```diff
@@ -1733,7 +1734,54 @@ def read_changes(
             .read(False, "default", read_options)
         )
 
-    def find_neighbors(self, embedding, col=None, k=10, filter=None, min_score=0):
+    def find_neighbors(
+        self,
+        embedding: List[Union[int, float]],
+        col: Optional[str] = None,
+        k: Optional[int] = 10,
+        filter: Optional[Union[Filter, Logic]] = None,
+        min_score: Optional[float] = 0
+    ) -> List[Tuple[float, List[Any]]]:
+        """
+        Finds the nearest neighbors for a given embedding in the vector database.
+
+        # Arguments
+            embedding: The target embedding for which neighbors are to be found.
```

Comment:
The name of the target embedding to be used for nearest neighbor (or similarity) search.

Reply:
This is not the name of the embedding, right?
It's the embedding_value or embedding_vector.
Maybe, you should make it more explicit to be clear to API users?

Reply:
As you mention in the definitive guide, embedding itself means a point in the embedding space, so embedding should be suffice.